staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/validating/metrics/metrics.go

```diff
@@ -120,3 +151,11 @@ func (m *ValidatingAdmissionPolicyMetrics) ObserveWarn(ctx context.Context, elap
 	m.policyCheck.WithContext(ctx).WithLabelValues(policy, binding, string(errorType), "warn").Inc()
 	m.policyLatency.WithContext(ctx).WithLabelValues(policy, binding, string(errorType), "warn").Observe(elapsed.Seconds())
 }
+
+// ObserveActiveBindings observes the count of active validation admission policy bindings.
+func (m *ValidatingAdmissionPolicyMetrics) ObserveActiveBindings(policyCount, allowCount, warnCount, denyCount int) {
+	m.policyActivePolicies.Set(float64(policyCount))
+	m.policyActiveBindingsEnforcement.WithLabelValues("allow").Set(float64(allowCount))
```

Comment:
Why not use audit which aligns with the value set?

Reply:
If we go with a lable inside active binding number, then use the same value as validationAction supports makes more sense.

Reply:
I can change this if you prefer.  I has been treating audit as orthogonal, and the ["warn","audit"] combination would result in the warn count being incremented but not the allow/audit count